docs/src/pages/api-reference/react.mdx

```diff
@@ -235,8 +240,8 @@ export const OurUploadDropzone = () => (
 | onBeforeUploadBegin    | `(files: File[]) => File[]`                          | No                      | Added in `v6.0` | callback function called before uploading starts. The files returned are the files that will be uploaded                    |
 | onUploadBegin          | function                                             | No                      | Added in `v5.4` | callback function for upload begin                                                                                          |
 | config                 | object                                               | No                      | Added in `v5.4` | object to pass additional configuration for dropzone                                                                        |
-| onDrop                 | (acceptedFiles: File[]) => void                      | No                      | Added in `v6.5` | Callback called when files are dropped or pasted.                                                                           |
 | disabled               | boolean                                              | No                      | Added in `v6.7` | Disables the dropzone.                                                                                                      |
+| onChange               | (acceptedFiles: File[]) => void                      | No                      | Added in `v6.8` | Callback called when files are dropped or pasted.                                                                           |
```

Comment:
Perhaps (and this circles back to your comment about how to handle deprecations in docs):

```
| onChange | (acceptedFiles: File[]) => void | No | Added in `v6.8` (`onDrop` in `v6.5-v-6.7`) | Callback called when files are dropped or pasted. |

Reply:
Yeah. I still think we should have versioned docs where you select the version you're on, but these inline comments are significantly less effort for now